### DISCUSSION TITLE
i need  help free tube seems to fail to build from the aur

### DISCUSSION THREAD

**Discussion by RAHUL12348697695848579** [FreeTubeApp/FreeTube#6402] [Q&A]
Hi there, I am an arch Linux user using cachyos and my FreeTube from the AUR seem to fail when I put it to build via pacman any help 
here are the logs 
:: Synchronizing package databases...
 cachyos-v3 is up to date
 cachyos-core-v3 is up to date
 cachyos-extra-v3 is up to date
 cachyos is up to date
 core is up to date
 extra is up to date
 multilib is up to date
:: Starting full system upgrade...
 there is nothing to do
:: Looking for PKGBUILD upgrades...
:: Looking for AUR upgrades...
:: Looking for devel upgrades...
:: Resolving dependencies...
:: Calculating conflicts...
:: Calculating inner conflicts...
:: packages not in the AUR: warp-terminal  xdm  xdman_gtk

Aur (1)       Old Version  New Version    Make Only
aur/freetube  0.22.0-1     0.22.1-1       No

:: Proceed to review? [Y/n]: Y

:: Downloading PKGBUILDs...
 PKGBUILDs up to date
 nothing new to review
fetching devel info...
==> Making package: freetube 0.22.1-1 (Tuesday 17 December 2024 07:25:54 PM)
==> Retrieving sources...
  -> Downloading v0.22.1-beta.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1874k    0 1874k    0     0   486k      0 --:--:--  0:00:03 --:--:--  750k
  -> Found freetube.desktop
  -> Found freetube.sh
==> Validating source files with sha256sums...
    v0.22.1-beta.tar.gz ... Passed
    freetube.desktop ... Passed
    freetube.sh ... Passed
==> Making package: freetube 0.22.1-1 (Tuesday 17 December 2024 07:26:00 PM)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found v0.22.1-beta.tar.gz
  -> Found freetube.desktop
  -> Found freetube.sh
==> Validating source files with sha256sums...
    v0.22.1-beta.tar.gz ... Passed
    freetube.desktop ... Passed
    freetube.sh ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting v0.22.1-beta.tar.gz with bsdtar
==> Starting prepare()...
==> Sources are ready.
freetube-0.22.1-1: parsing pkg list...
==> Making package: freetube 0.22.1-1 (Tuesday 17 December 2024 07:26:04 PM)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Removing existing $pkgdir/ directory...
==> Starting build()...
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager

yarn install v1.22.22
[1/4] Resolving packages...
[2/4] Fetching packages...
error Error: https://registry.yarnpkg.com/cssesc/-/cssesc-3.0.0.tgz: Request failed "304 Not Modified"
    at ResponseError.ExtendableBuiltin (/home/rahul/.cache/node/corepack/v1/yarn/1.22.22/lib/cli.js:696:66)
    at new ResponseError (/home/rahul/.cache/node/corepack/v1/yarn/1.22.22/lib/cli.js:802:124)
    at Request.<anonymous> (/home/rahul/.cache/node/corepack/v1/yarn/1.22.22/lib/cli.js:66750:16)
    at Request.emit (node:events:513:28)
    at module.exports.Request.onRequestResponse (/home/rahul/.cache/node/corepack/v1/yarn/1.22.22/lib/cli.js:142287:10)
    at ClientRequest.emit (node:events:513:28)
    at HTTPParser.parserOnIncomingClient (node:_http_client:710:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:117:17)
    at TLSSocket.socketOnData (node:_http_client:552:22)
    at TLSSocket.emit (node:events:513:28)
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
==> ERROR: A failure occurred in build().
    Aborting...
error: failed to build 'freetube-0.22.1-1': 
error: packages failed to build: freetube-0.22.1-1
System update failed: 
   0: Command failed: `/usr/bin/paru --pacman pacman -Syu`
   1: `/usr/bin/paru` failed: exit status: 1

Location:
   src/steps/os/archlinux.rs:53
any help?

**Comment by absidue** [comment]
The AUR packages are unofficial and not maintained by the FreeTube team, so we can't help you with problems specific to the AUR packages. Please contact that maintainers of the AUR package that you are using.
